controllers/common.go

```diff
@@ -530,7 +621,72 @@ func (r *MultiClusterHubReconciler) ensureMCESubscription(ctx context.Context, m
 		// Sub is nil so create a new one
 		mceSub = multiclusterengine.NewSubscription(multiClusterHub, subConfig, overrides, utils.IsCommunityMode())
 	} else if multiclusterengine.CreatedByMCH(mceSub, multiClusterHub) {
-		result, err := r.ensurePullSecret(multiClusterHub, multiclusterengine.Namespace().Name)
+		result, err := r.ensurePullSecret(multiClusterHub, mceNamespace.Name)
+		if result != (ctrl.Result{}) {
+			return result, err
+		}
+		result, err = r.ensureOperatorGroup(multiClusterHub, multiclusterengine.OperatorGroup())
+		if result != (ctrl.Result{}) {
+			return result, err
+		}
+	}
+
+	// Apply MCE sub
+	calcSub := multiclusterengine.RenderSubscription(mceSub, subConfig, overrides, ctlSrc, utils.IsCommunityMode())
+
+	force := true
+	err = r.Client.Patch(ctx, calcSub, client.Apply, &client.PatchOptions{Force: &force, FieldManager: "multiclusterhub-operator"})
+	if err != nil {
+		r.Log.Info(fmt.Sprintf("Error applying subscription: %s", err.Error()))
+		return ctrl.Result{Requeue: true}, err
+	}
+	return ctrl.Result{}, nil
+}
+
+func (r *MultiClusterHubReconciler) ensureHostedMCESubscription(ctx context.Context, multiClusterHub *operatorv1.MultiClusterHub) (ctrl.Result, error) {
```

Comment:
We decided it would be better if only the main MCH handled the MCE subscription so hosted MCH's aren't affecting the same resources. Managing the MCE subscription is not necessary and could cause issues.